docs/source/reference/indexing.rst

```diff
@@ -23,6 +23,8 @@ Properties
    Index.names
    Index.ndim
    Index.empty
+   Index.T
+   Index.transpose
```

Comment:
Ah, then let's exclude `transpose` in the API documentation to be consistent.

Reply:
okay i just excluded them 👍